bot/cogs/source.py

```diff
@@ -91,6 +99,9 @@ async def build_embed(self, link: str, source_object: Union[HelpCommand, Command
                 description = source_object.help
 
             title = source_object.qualified_name
+        elif isinstance(source_object, str):
+            title = f"Tag: {source_object.split('/')[-1].split('.')[0]}"
```

Comment:
This parsing can be avoided:

1. Implement my other suggestion regarding the tag cache
2. Have the converter simply return the tag name
3. Make `get_source_link` responsible for fetching the file path for the tag
4. In here you will have the original tag name, so no parsing needed!